### PR TITLE
[0.17.3] Add analytical risk as arm constituent

### PR DESCRIPTION
**Pull request by koenderks**
- For ADR, we added the analytical risk as a constituent of the audit risk model.
- Also adds confidence intervals around proportions for Benford's law.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-Version: 0.17.2
-Date: 2023-02-08
+Version: 0.17.3
+Date: 2023-06-29
 Author: Koen Derks, Jacques de Swart, Eric-Jan Wagenmakers, Jan Wille, and Ruud Wetzels
 Website: https://github.com/jasp-stats/jaspAudit
 Maintainer: Koen Derks <k.derks@nyenrode.nl>
 Description: Statistical methods for auditing.
-License: GPL (>= 2)
+License: GPL (>= 3)
```
Comment (vandenman) [inline DESCRIPTION:10]: why this change? this should be standardized within JASP.

File: R/auditClassicalBenfordsLaw.R [hunk 2]
```diff
@@ -209,6 +220,7 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...) {
   tb$dependOn(options = "bayesFactorType")
   tb$addColumnInfo(name = "test", title = "", type = "string")
   tb$addColumnInfo(name = "N", title = "n", type = "integer")
+  tb$addColumnInfo(name = "mad", title = "MAD", type = "number")
```
Comment (vandenman) [inline R/auditClassicalBenfordsLaw.R:223]:
```suggestion
  tb$addColumnInfo(name = "mad", title = gettext("MAD"), type = "number")
```

File: inst/qml/common/planning/AuditRiskModel.qml [hunk 3]
```diff
@@ -83,4 +91,33 @@ Group
...
+			{ label: qsTr("Low"), 		value: "low"},
+			{ label: qsTr("Custom"), 	value: "custom"}
+		]
+		info:					qsTr("A category or probability for the analytical risk. Analytical risk is defined as the the risk that a material misstatement will not be discovered by the auditor through analytical procedures.")
+	}
+
+	PercentField
+	{
+		name: 					"carCustom"
+		defaultValue:			car.value == "high" ? 100 : (car.value == 'medium' ? 50 : (car.value == "low" ? 25 : 100))
```
Comment (vandenman) [inline inst/qml/common/planning/AuditRiskModel.qml:118]:
```suggestion
		defaultValue:			car.value === "high" ? 100 : (car.value === 'medium' ? 50 : (car.value === "low" ? 25 : 100))
```

File: inst/qml/common/planning/AuditRiskModel.qml [hunk 4]
```diff
@@ -83,4 +91,33 @@ Group
...
+		min: 					1
+		enabled: 				car.value == "custom"
```
Comment (vandenman) [inline inst/qml/common/planning/AuditRiskModel.qml:120]:
```suggestion
		enabled: 				car.value === "custom"
```

**Review by vandenman — CHANGES_REQUESTED** [review]
some very minor stuff, see below

File: DESCRIPTION (hunk 1 above)
Comment (koenderks) [reply DESCRIPTION:10]: It contradicted this line in `description.qml`:

```
license				: "GPL (>= 3)"
```

File: R/auditClassicalBenfordsLaw.R (hunk 2 above)
Comment (koenderks) [reply R/auditClassicalBenfordsLaw.R:223]: Thanks!

File: inst/qml/common/planning/AuditRiskModel.qml (hunk 4 above)
Comment (koenderks) [reply inst/qml/common/planning/AuditRiskModel.qml]: Learned something new!

**Review by vandenman — APPROVED** [review]
